extensions/wikia/ArticleVideo/scripts/featured-video.jwplayer.instant.js

```diff
@@ -35,6 +35,7 @@ require([
 		slotTargeting = {
 			plist: recommendedPlaylist
 		},
+		responseTimeout = 2000,
```

Comment:
Shouldn't we use `wgAdDriverDelayTimeout` instead of hardcoded value?

Reply:
Hmmm. Maybe. I think that it was not used in the previous code.

Reply:
you're right. We can leave it as it is for now,